store/root.js

```diff
@@ -137,23 +142,15 @@ class Root extends Store {
       return
     }
 
-    // Compare strings and numbers
-    // since we do not really know what type of ID the API exposes
-    // eslint-disable-next-line eqeqeq
-    const hasTargetId = (dataProvider) => id == dataProvider.id
+    const dataProviderInt = parseInt(id, 10)
```

Comment:
Why don't you like it? I added a function like this because the API really can migrate IDs to UUIDs (@samuelpearce tends to say that, let me know if I am wrong), what will make use to change this once again.

Reply:
I am not saying that I don't like it. I don't think about these things honestly. I found it better to convert it because I spared one or two lines of code. That's all. If dataProvider changes to UUID I am happy to change it. :)